src/sentry/web/frontend/debug/mail.py

```diff
@@ -265,6 +266,7 @@ def alert(request):
 
     event_manager = EventManager(data)
     event_manager.normalize()
+    event = event_manager.save(project.id)
```

Comment:
Wait, doesn't this start a whole lot of operations checking for hashes, user feedback, etc?

Reply:
Discussed in slack: This endpoint is not used in production or staging.